thejamesgore/Foxtons-Dashboard
# feat: Add property and user controllers and routes, update server.js

## Description 🚀

This pull request adds property and user controllers and routes to the project, and updates server.js to use these controllers and routes.

## Changes Made 👉
- Created property.controller.js and user.controller.js files to handle requests to the properties and users endpoints, respectively.
- Created property.routes.js and user.routes.js files to define the routes for the properties and users endpoints, respectively.
- Updated server.js to use the property and user routes and controllers.
- Added three more tests 

## Reason for Changes 🔍
By adding property and user controllers and routes to our codebase, we can better handle requests to the properties and users endpoints, and improve the overall structure and organization of our application.

## How to Test
To test these changes, you can follow these steps:
- Clone this branch to your local machine
- Install the necessary dependencies with `npm install`
- Start the application with `npm start`
- Verify that the application starts without any errors and that requests to the properties and users endpoints return the expected responses.

## Checklist

## Related Issues
None.

## Other Notes
None.
